Codeinwp/optimole-wp
# feat: add cli command for clearing cache

### All Submissions:

 

### Changes proposed in this Pull Request:
 
This adds a new CLI command to clear cache to Optimole. The command is under the settings and can be used like:

`usage: wp optimole setting clear_cache --type=<type>`

`type` defaults to images, and can be changed to assets.

Closes #607.

### How to test the changes in this Pull Request:

1. Try to clear cache using UI and CLI command and make sure they run smoothly.
2. Keep in mind, you can't clear cache for a particular type (images/assets) more than once every 5 minute.

### Other information:
